src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -32,6 +32,12 @@ public class DefaultModelMetadata : ModelMetadata
         private ModelPropertyCollection _properties;
         private bool? _validateChildren;
         private ReadOnlyCollection<object> _validatorMetadata;
+        private bool _invokedDescription = false;
+        private string _cachedDescription = null;
+        private bool _invokedDisplayName = false;
+        private string _cachedDisplayName = null;
+        private bool _invokedPlaceholder = false;
+        private string _cachedPlaceholder = null;
```

Comment:
Ah, ok. I wasn't aware that they have long lives. I suppose now I am just not sure of what @rynowak means about storing `Placeholder` in a local.